docs/concepts/models/sql_models.md

```diff
@@ -1,37 +1,40 @@
 # SQL models
 
-SQL models are the main types of models used by SQLMesh. SQL models consist of three sections:
+SQL models are the main type of models used by SQLMesh. These models can be defined using either SQL or Python.
 
-* `MODEL` DDL
-* Optional sql statements
-* The model's `SELECT` statement
+## SQL-based definition
 
-SQL models are designed to look and feel like you're simply using SQL, but they can be customized for advanced use cases.
+The SQL-based definition of SQL models is the most common one, and consists of the following sections:
```

Comment:
it should say SQL models can be defined in pure SQL, without YAML etc, why did you get rid of the simply sql section?

Reply:
It's just the git diff here; the "simply SQL" part is just below this section, I haven't removed it.